packages/core/helper-plugin/src/components/ConfirmDialog.tsx

```diff
@@ -0,0 +1,187 @@
+import React from 'react';
+
+import {
+  Button,
+  ButtonProps,
+  Box,
+  BoxProps,
+  Dialog,
+  DialogBody,
+  DialogFooter,
+  Flex,
+  Typography,
+  DialogBodyProps,
+  FlexProps,
+  DialogProps,
+} from '@strapi/design-system';
+import { ExclamationMarkCircle, Trash } from '@strapi/icons';
+import { useIntl } from 'react-intl';
+
+export interface RootProps extends Partial<FooterProps> {
+  children: BoxProps['children'];
```

Comment:
```suggestion
export interface RootProps extends Partial<FooterProps>, Pick<BoxProps, 'children'> {
```

I think you can also do this?